function-support.html

```diff
@@ -321,6 +319,43 @@ <h2 id="accents">Accents</h2>
 			<td></td>
 		</tr>
 	</table>
+	
+	<h4 id="accents-inside-\text…}">Accents inside \text{…}</h4>
+
+	<table>
+		<tr>
+			<td>\(\text{\&apos;{a}}\)</td>
+			<td>\&apos;{a}</td>
+			<td>\(\text{\~{a}}\)</td>
+			<td>\~{a}</td>
+			<td>\(\text{\.{a}}\)</td>
+			<td>\.{a}</td>
+			<td>\(\text{\H{a}}\)</td>
+			<td>\H{a}</td>
+		</tr>
+		<tr>
+			<td>\(\text{\`{a}}\)</td>
+			<td>\`{a}</td>
+			<td>\(\text{\={a}}\)</td>
+			<td>\={a}</td>
+			<td>\(\text{\&quot;{a}}\)</td>
+			<td>\&quot;{a}</td>

```

Comment:
I was surprised to see `&quot;` here.  `"` should be safe here, no?

Reply:
Software again. It's written to always replace quotation marks with either `&quot;` or `<q>` tags.